apps/wallet-dashboard/components/SupplyIncreaseVestingOverview.tsx

```diff
@@ -0,0 +1,149 @@
+    } = useStakeDialog();
+
+    const formattedLastPayoutExpirationTime = useCountdownByTimestamp(
+        Number(nextPayout?.expirationTimestampMs),
+        false,
+        false,
+    );
+    const [formattedNextPayout, nextPayoutSymbol, nextPayoutResult] = useFormatCoin(
+        nextPayout?.amount,
+        IOTA_TYPE_ARG,
+    );
+
+    const [formattedAvailableStaking, availableStakingSymbol] = useFormatCoin(
+        vestingSchedule.availableStaking,
+        IOTA_TYPE_ARG,
+    );
+
+    const showSupplyIncreaseVestingOverview =
+        vestingMapped.length > 0 || vestingStakedMapped.length > 0;
+
+    function handleOnSuccess(digest: string): void {
+        iotaClient
+            .waitForTransaction({
+                digest,
+            })
+            .then(() => {
+                queryClient.invalidateQueries({
+                    queryKey: ['get-timelocked-staked-objects', account?.address],
+                });
+                queryClient.invalidateQueries({
+                    queryKey: [
+                        'get-all-owned-objects',
+                        account?.address,
+                        {
+                            StructType: TIMELOCK_IOTA_TYPE,
+                        },
+                    ],
+                });
+            });
+    }
+
+    return showSupplyIncreaseVestingOverview ? (
+        <div style={{ gridArea: 'vesting' }} className="with-vesting flex grow overflow-hidden">
+            <Panel>
+                <Title title="Vesting" />
+                <div className="flex h-full w-full items-center gap-md p-md--rs">
+                    <div className="w-1/2">
+                        <Card type={CardType.Filled} isHoverable>
+                            <CardBody
+                                title=""
+                                subtitle={
+                                    <LabelText
+                                        size={LabelTextSize.Large}
+                                        label="Next reward"
+                                        text={
+                                            nextPayoutResult.isPending
+                                                ? '-'
+                                                : `${formattedNextPayout}   `
+                                        }
+                                        supportingLabel={nextPayoutSymbol}
+                                    />
+                                }
+                            />
+                            <CardAction
+                                type={CardActionType.Button}
+                                buttonType={ButtonType.Ghost}
+                                title={formattedLastPayoutExpirationTime}
+                                icon={<SvgClock />}
+                            />
+                        </Card>
+                    </div>
+                    <div className="w-1/2">
+                        <Card type={CardType.Filled} isHoverable>
```

Comment:
same with this one :pray: 
```suggestion
                        <Card type={CardType.Filled} >
```